src/main/java/io/supertokens/authRecipe/AuthRecipe.java

```diff
@@ -213,6 +213,34 @@ private static void deleteNonAuthRecipeUser(AppIdentifierWithStorage
         });
     }
 
+    public static boolean deleteNonAuthRecipeUser(TenantIdentifierWithStorage
```

Comment:
we need a way to enforce that this function is modified as well whenever the above `deleteNonAuthRecipeUser` function is modified. Either through a test or some code refactor.

Reply:
deleteUserIdFromTenantId test will ensure this.